client/package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "@mui/icons-material": "^5.2.0",
     "@mui/material": "^5.2.1",
+    "@reduxjs/toolkit": "^1.6.2",
```

Comment:
Honestly saying I would like us to proceed without meta tools for redux in this course. Let's train with pure redux and then you can play around with helper libraries.